sql/001-create-table-and-load-data.sql

```diff
@@ -1,10 +1,18 @@
 DROP TABLE IF EXISTS players;
 CREATE TABLE players
 (
-    id int unsigned AUTO_INCREMENT, name VARCHAR(20) NOT NULL, number int(5) NOT NULL, prefecture VARCHAR(10) NOT NULL,
+    id int unsigned AUTO_INCREMENT, name VARCHAR(20) NOT NULL, position VARCHAR(15) NOT NULL, number int(5) NOT NULL, prefecture VARCHAR(10) NOT NULL,
```

Comment:
ここはワンライナーではなく改行した方が見やすいですね👀
```suggestion
    id int unsigned AUTO_INCREMENT, 
    name VARCHAR(20) NOT NULL, 
    position VARCHAR(15) NOT NULL, 
    number int(5) NOT NULL, 
    prefecture VARCHAR(10) NOT NULL,
```

Reply:
`number`というカラム名は抽象的すぎるので`uniform_number`にした方が良いかも...

Reply:
a3e82f
お疲れ様です！
uniform_numberに修正いたしました！